README.md

```diff
@@ -42,7 +40,16 @@ In addition to the apps above, IPFS GUIs can be considered to include aspects of
 - **[ipfs-shipyard/ipfs-ui-style-guide](https://www.github.com/ipfs-shipyard/ipfs-ui-style-guide)**: UI style guide for IPFS apps
 - **[ipfs-shipyard/pinning-service-compliance](https://www.github.com/ipfs-shipyard/pinning-service-compliance)**: A test suite to help see which pinning service providers are correctly implementing the [pinning services spec](https://ipfs.github.io/pinning-services-api-spec/).
 - **[ipfs-shipyard/js-pinning-service-http-client](https://github.com/ipfs-shipyard/js-pinning-service-http-client)**: A pinning service client for the browser and node
+- **[ipfs-shipyard/ipfs-dag-builder-vis](https://github.com/ipfs-shipyard/ipfs-dag-builder-vis)**: A tool for creating & modifying IPFS DAG structures visually. See https://dag.ipfs.tech/ 
+
+### Other GUI projects
+
+In addition to the apps & repos above, other relevant GUI related tools/apps that are not owned by the IPFS GUI & Tools team are:
+
 - **[kubo/dir-index-html](https://github.com/ipfs/go-ipfs/tree/master/assets/dir-index-html)**: Directory listing HTML for kubo (go-ipfs) gateways
+- **[filecoin-station/filecoin-station](https://github.com/filecoin-station/filecoin-station)**: Filecoin Station connects your computer’s idle resources to the Filecoin network and rewards you with FIL. Taking part is simple, just launch the app and start earning.
```

Comment:
@SgtPooki  Why is this here? IPFS Stewards don't own this project and https://www.filstation.app/ does not mention IPFS even once.

Reply:
Because it's an

It's a GUI app thats related to ipfs and it's explicitly in the section denoted as not owned by us.

> other relevant GUI related tools/apps that are not owned by the IPFS GUI & Tools team

Also, there are some plans to share components between the apps. I think it makes sense to call out other apps ipfs gui devs (or external contributors) might be interested in.